<!DOCTYPE html>
<html lang="en-US">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Martin Fung&#39;s Page</title>
  <meta name="description" content="">
  <link rel="canonical" href="http://localhost:4000/page6/">
  <link rel="alternate" type="application/rss+xml" title="Martin Fung&#39;s Page Feed"
    href="http://localhost:4000/feed.xml">
  
  <link rel="shortcut icon" href="/favicon.png" type="image/png" />
  
  <!-- Styles -->
  <link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700,700i%7CNoto+Serif:400,400i,700,700i&display=swap" rel="stylesheet">
  <link href="/assets/css/style.css" rel="stylesheet">
</head>

<body>

  <div id="page" class="site">
    <div class="inner">
      <h1 style="text-align: center; padding: 1em 0 0.3em">Martin Fung's Page
</h1>

<h2 style="text-align: center; margin: 0">
    <span>| <a href="/">Home</a></span>
    
    
    
    <span>|</span>
    <a class="" href="/tags/">Blog</a>
    
    
    
    <span>|</span>
    <a class="" href="/cv">CV</a>
    
    <span>|</span>
</h2>
<hr>

      <main class="main-content fadeInDown delay_075s">

  <article class="post" style="border: none; margin-bottom: 5em">
  <h2>Hello!</h2>
  <ul>
  <li>I'm Martin Fung.</li>
  <li>I am a biologist from Hong Kong.</li>
  <li><a href="https://pubmed.ncbi.nlm.nih.gov/?term=To+Sing+Fung%5Bau%5D+or+24987391%5Bpmid%5D">Click here for my PubMed search page</a></li>
  <li><a href="https://scholar.google.com/citations?user=igL5v0QAAAAJ">Click here for my Google Scholar page</a></li>
  </article>

  <div style="
  background:#f0f0f0;
  text-align:center;
  padding: 1em 0 1em;">
  <h2 style="margin: 0">
  - Recent posts -</h2>
  </div>
  
  
  <article class="post">
    <header class="post-header">
      <time class="post-date" datetime="2021-06-17">June 17, 2021</time>
      <h2 class="post-title"><a href="/Biopython-Basic-seq-operations" rel="bookmark">Biopython Basic seq operations</a></h2>
      <div class="post-meta">
        <span
          class="post-tags"> Tags: 
          <a href="/tags/#python-learn" rel="tag">python-learn</a></span>
      </div><!-- .post-meta -->
      
    </header><!-- .post-header -->
    <div class="post-content">
      
      <h2 id="comp-and-rev_comp">comp and rev_comp</h2>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">Bio.Seq</span> <span class="kn">import</span> <span class="n">Seq</span>
<span class="n">dna</span> <span class="o">=</span> <span class="n">Seq</span><span class="p">(</span><span class="s">"AGGGCCCTTTA"</span><span class="p">)</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">[::</span><span class="o">-</span><span class="mi">1</span><span class="p">])</span> <span class="c1"># reverse
</span><span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">.</span><span class="n">complement</span><span class="p">())</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">.</span><span class="n">reverse_complement</span><span class="p">())</span>
</code></pre></div></div>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>ATTTCCCGGGA
TCCCGGGAAAT
TAAAGGGCCCT
</code></pre></div></div>

<h2 id="parse-fasta-file">parse fasta file</h2>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">Bio</span> <span class="kn">import</span> <span class="n">SeqIO</span>
<span class="k">for</span> <span class="n">record</span> <span class="ow">in</span> <span class="n">SeqIO</span><span class="p">.</span><span class="n">parse</span><span class="p">(</span><span class="s">"ibv.fasta"</span><span class="p">,</span><span class="s">"fasta"</span><span class="p">):</span>
    <span class="k">print</span><span class="p">(</span><span class="n">record</span><span class="p">.</span><span class="nb">id</span><span class="p">)</span>
    <span class="k">print</span><span class="p">(</span><span class="nb">len</span><span class="p">(</span><span class="n">record</span><span class="p">.</span><span class="n">seq</span><span class="p">))</span>
    <span class="k">print</span><span class="p">(</span><span class="nb">repr</span><span class="p">(</span><span class="n">record</span><span class="p">.</span><span class="n">seq</span><span class="p">))</span>
</code></pre></div></div>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>DQ001339.1
27611
Seq('ACTTAAGATAGATATTAATATATATCTATTATACTAGCCTTGCGCTAGATTTTT...GCA')
</code></pre></div></div>

<h2 id="parse-genbank-file">parse genbank file</h2>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">Bio</span> <span class="kn">import</span> <span class="n">SeqIO</span>
<span class="k">for</span> <span class="n">record</span> <span class="ow">in</span> <span class="n">SeqIO</span><span class="p">.</span><span class="n">parse</span><span class="p">(</span><span class="s">"sars2.gb"</span><span class="p">,</span><span class="s">"genbank"</span><span class="p">):</span>
    <span class="k">print</span><span class="p">(</span><span class="n">record</span><span class="p">.</span><span class="nb">id</span><span class="p">)</span>
    <span class="k">print</span><span class="p">(</span><span class="nb">len</span><span class="p">(</span><span class="n">record</span><span class="p">.</span><span class="n">seq</span><span class="p">))</span>
    <span class="k">print</span><span class="p">(</span><span class="nb">repr</span><span class="p">(</span><span class="n">record</span><span class="p">.</span><span class="n">seq</span><span class="p">))</span>
</code></pre></div></div>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>NC_045512.2
29903
Seq('ATTAAAGGTTTATACCTTCCCAGGTAACAAACCAACCAACTTTCGATCTCTTGT...AAA')
</code></pre></div></div>

<h2 id="gc-content">GC content</h2>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">Bio.Seq</span> <span class="kn">import</span> <span class="n">Seq</span>
<span class="kn">from</span> <span class="nn">Bio.SeqUtils</span> <span class="kn">import</span> <span class="n">GC</span>
<span class="n">dna</span> <span class="o">=</span> <span class="n">Seq</span><span class="p">(</span><span class="s">"GaTCGaTGGGCCTaTATAGGATCGaaaaTCGC"</span><span class="p">)</span>
<span class="k">print</span> <span class="p">(</span><span class="sa">f</span><span class="s">"""</span><span class="se">\
</span><span class="s">A: </span><span class="si">{</span><span class="n">dna</span><span class="p">.</span><span class="n">count</span><span class="p">(</span><span class="s">"A"</span><span class="p">)</span><span class="si">}</span><span class="s">
a: </span><span class="si">{</span><span class="n">dna</span><span class="p">.</span><span class="n">count</span><span class="p">(</span><span class="s">"a"</span><span class="p">)</span><span class="si">}</span><span class="s">
T: </span><span class="si">{</span><span class="n">dna</span><span class="p">.</span><span class="n">count</span><span class="p">(</span><span class="s">"T"</span><span class="p">)</span><span class="si">}</span><span class="s">
G: </span><span class="si">{</span><span class="n">dna</span><span class="p">.</span><span class="n">count</span><span class="p">(</span><span class="s">"G"</span><span class="p">)</span><span class="si">}</span><span class="s">
C: </span><span class="si">{</span><span class="n">dna</span><span class="p">.</span><span class="n">count</span><span class="p">(</span><span class="s">"C"</span><span class="p">)</span><span class="si">}</span><span class="s">
GC </span><span class="si">{</span><span class="n">GC</span><span class="p">(</span><span class="n">dna</span><span class="p">)</span><span class="si">}</span><span class="s">%</span><span class="se">\
</span><span class="s">"""</span><span class="p">)</span>
<span class="c1"># .count() is sensitive to mixed case
# .GC() is not senstitive
</span></code></pre></div></div>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>A: 3
a: 7
T: 7
G: 9
C: 6
GC 46.875%
</code></pre></div></div>

<h2 id="slicing-a-sequence">slicing a sequence</h2>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">Bio.Seq</span> <span class="kn">import</span> <span class="n">Seq</span>
<span class="n">dna</span> <span class="o">=</span> <span class="n">Seq</span><span class="p">(</span><span class="s">"GATCGATGGGCCTATATAGGATCGAAAATCGC"</span><span class="p">)</span>
<span class="c1">#          0123456789
</span><span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">[</span><span class="mi">3</span><span class="p">:</span><span class="mi">9</span><span class="p">])</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">[</span><span class="mi">0</span><span class="p">::</span><span class="mi">3</span><span class="p">])</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">[</span><span class="mi">1</span><span class="p">::</span><span class="mi">3</span><span class="p">])</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">[</span><span class="mi">2</span><span class="p">::</span><span class="mi">3</span><span class="p">])</span>
</code></pre></div></div>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>CGATGG
GCTGTAGTAAG
AGGCATGCATC
TAGCTAAGAC
</code></pre></div></div>

<h2 id="concatenate-sequence">concatenate sequence</h2>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">Bio.Seq</span> <span class="kn">import</span> <span class="n">Seq</span>
<span class="n">l_seq</span> <span class="o">=</span> <span class="p">[</span><span class="n">Seq</span><span class="p">(</span><span class="s">"ACGT"</span><span class="p">),</span> <span class="n">Seq</span><span class="p">(</span><span class="s">"AACC"</span><span class="p">),</span> <span class="n">Seq</span><span class="p">(</span><span class="s">"GGTT"</span><span class="p">)]</span>
<span class="n">concat</span> <span class="o">=</span> <span class="n">Seq</span><span class="p">(</span><span class="s">""</span><span class="p">)</span>
<span class="k">for</span> <span class="n">s</span> <span class="ow">in</span> <span class="n">l_seq</span><span class="p">:</span>
    <span class="n">concat</span> <span class="o">+=</span> <span class="n">s</span>
<span class="k">print</span> <span class="p">(</span><span class="n">concat</span><span class="p">)</span>
</code></pre></div></div>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>ACGTAACCGGTT
</code></pre></div></div>

<h2 id="join-sequence">join sequence</h2>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">Bio.Seq</span> <span class="kn">import</span> <span class="n">Seq</span>
<span class="n">l_seq</span> <span class="o">=</span> <span class="p">[</span><span class="n">Seq</span><span class="p">(</span><span class="s">"ACGT"</span><span class="p">),</span> <span class="n">Seq</span><span class="p">(</span><span class="s">"AACC"</span><span class="p">),</span> <span class="n">Seq</span><span class="p">(</span><span class="s">"GGTT"</span><span class="p">)]</span>
<span class="k">print</span> <span class="p">(</span><span class="n">Seq</span><span class="p">(</span><span class="s">""</span><span class="p">).</span><span class="n">join</span><span class="p">(</span><span class="n">l_seq</span><span class="p">))</span>
<span class="k">print</span> <span class="p">(</span><span class="n">Seq</span><span class="p">(</span><span class="s">"nnnnn"</span><span class="p">).</span><span class="n">join</span><span class="p">(</span><span class="n">l_seq</span><span class="p">))</span>
</code></pre></div></div>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>ACGTAACCGGTT
ACGTnnnnnAACCnnnnnGGTT
</code></pre></div></div>

<h2 id="upper-and-lower">upper and lower</h2>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">Bio.Seq</span> <span class="kn">import</span> <span class="n">Seq</span>
<span class="n">dna</span> <span class="o">=</span> <span class="n">Seq</span><span class="p">(</span><span class="s">"GATCGATGGGCCTATATAGGATCGAAAATCGC"</span><span class="p">)</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">.</span><span class="n">lower</span><span class="p">())</span>
<span class="k">print</span> <span class="p">(</span><span class="s">"GATC"</span> <span class="ow">in</span> <span class="n">dna</span><span class="p">)</span>
<span class="k">print</span> <span class="p">(</span><span class="s">"GATC"</span> <span class="ow">in</span> <span class="n">dna</span><span class="p">.</span><span class="n">lower</span><span class="p">())</span> <span class="c1"># case matching
</span></code></pre></div></div>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>gatcgatgggcctatataggatcgaaaatcgc
True
False
</code></pre></div></div>

<h2 id="transcribe-and-translate">transcribe and translate</h2>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">Bio.Seq</span> <span class="kn">import</span> <span class="n">Seq</span>
<span class="n">dna</span> <span class="o">=</span> <span class="n">Seq</span><span class="p">(</span><span class="s">"ATGGCCATTGTAATGGGCCGCTGAAAGGGTGCCCGATAG"</span><span class="p">)</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">.</span><span class="n">transcribe</span><span class="p">())</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">.</span><span class="n">translate</span><span class="p">())</span>
</code></pre></div></div>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>AUGGCCAUUGUAAUGGGCCGCUGAAAGGGUGCCCGAUAG
MAIVMGR*KGAR*
</code></pre></div></div>

<h2 id="translate-parameters">translate parameters</h2>

<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="kn">from</span> <span class="nn">Bio.Seq</span> <span class="kn">import</span> <span class="n">Seq</span>
<span class="n">dna</span> <span class="o">=</span> <span class="n">Seq</span><span class="p">(</span><span class="s">"ATGGCCATTGTAATGGGCCGCTGAAAGGGTGCCCGATAG"</span><span class="p">)</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">.</span><span class="n">translate</span><span class="p">())</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">.</span><span class="n">translate</span><span class="p">(</span><span class="n">to_stop</span><span class="o">=</span><span class="bp">True</span><span class="p">))</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">.</span><span class="n">translate</span><span class="p">(</span><span class="n">table</span><span class="o">=</span><span class="s">"Vertebrate Mitochondrial"</span><span class="p">))</span>
<span class="k">print</span> <span class="p">(</span><span class="n">dna</span><span class="p">.</span><span class="n">translate</span><span class="p">(</span><span class="n">table</span><span class="o">=</span><span class="s">"Vertebrate Mitochondrial"</span><span class="p">,</span> <span class="n">to_stop</span><span class="o">=</span><span class="bp">True</span><span class="p">))</span>
</code></pre></div></div>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>MAIVMGR*KGAR*
MAIVMGR
MAIVMGRWKGAR*
MAIVMGRWKGAR
</code></pre></div></div>


      
    </div><!-- .post-content -->
  </article><!-- .post -->
  
  <article class="post">
    <header class="post-header">
      <time class="post-date" datetime="2021-06-16">June 16, 2021</time>
      <h2 class="post-title"><a href="/research-t7-promoter-primer-design" rel="bookmark">T7 promoter flanking sequence</a></h2>
      <div class="post-meta">
        <span
          class="post-tags"> Tags: 
          <a href="/tags/#Research" rel="tag">Research</a></span>
      </div><!-- .post-meta -->
      
    </header><!-- .post-header -->
    <div class="post-content">
      
      <p><img src="https://raw.githubusercontent.com/tosingfung/images/master/image-20210616150924753.png" alt="image-20210616150924753" /></p>

<h4 id="the-4-to-8-downstream-sequence-affect-the-transcription-efficiency-of-t7-promoter">The +4 to +8 downstream sequence affect the transcription efficiency of T7 promoter.</h4>

<p>A <a href="https://www.nature.com/articles/s42003-020-01167-x">study</a> has shown that +4 to +8 nucleotides downstream of T7 significantly affect transcription efficiency. Taken this into consideration, together with the user guide from <a href="https://www.thermofisher.cn/content/dam/LifeTech/global/life-sciences/CellCultureandTransfection/pdfs/Lipofectamine_MessengerMAX_man.pdf">Lipofectamine MessengerMAX</a>, the PCR forward primer should be (the reverse primer should contain a polyA tail):</p>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>GC TAATACGACTCACTATA GGG ATAAT gccacc ATG...
   |                 |         |
   T7                Tss       kozak
</code></pre></div></div>


      
    </div><!-- .post-content -->
  </article><!-- .post -->
  
  <article class="post">
    <header class="post-header">
      <time class="post-date" datetime="2021-06-16">June 16, 2021</time>
      <h2 class="post-title"><a href="/research-lentivirus-protocol-capacity" rel="bookmark">lentivirus protocol and capacity</a></h2>
      <div class="post-meta">
        <span
          class="post-tags"> Tags: 
          <a href="/tags/#Research" rel="tag">Research</a></span>
      </div><!-- .post-meta -->
      
    </header><!-- .post-header -->
    <div class="post-content">
      
      <h2 id="a-simplified-lentivirus-protocol">A simplified lentivirus protocol</h2>

<ol>
  <li>Plate HEK293T, 12-well plate</li>
  <li>Transfect:
    <ul>
      <li>0.50 μg psPAX2</li>
      <li>0.25 μg pMD2.G</li>
      <li>1.25 μg pLVX</li>
    </ul>
  </li>
  <li>Change 550 μl DMEM / 10% FBS</li>
  <li>48 hpt, harvest sp, keep 4 degree</li>
  <li>Add 550 μl DMEM / 10% FBS</li>
  <li>72 hpt, harvest and pool</li>
  <li>Spin down, aliquot, -80 degree</li>
  <li>Transduction (12-well plate)</li>
  <li>Use 100-300 μl lentivirus</li>
  <li>Change medium (optional)</li>
  <li>Harvest after 48-72 h</li>
</ol>

<h2 id="cloningpackaging-capacity">Cloning/packaging capacity</h2>

<p>Wild-type lentiviruses contain ~9.7 kb of genome including both LTRs. <strong>Artificially creating a genome larger than this will result in unstable viral particles and a dramatic drop in viral titer</strong>. For recombinant lentiviruses such as those generated using Lenti-X systems, much of the viral genome has been replaced with other useful sequences such as selection markers or fluorescent proteins, but enough space remains for cloning transgenes. Since each pLVX vector contains different useful sequences, the available space for cloning your transgene varies. To determine the maximum recommended transgene size, consult your pLVX vector map and determine the position of the end of the 3’ LTR (the end of the 5’ LTR is always at position 1), then subtract that from 9.7 kb. For example: the 3’ LTR of pLVX-Puro ends at 5.4 kb, so ~4.3 kb of space remains for you to clone in your gene (<a href="https://www.takarabio.com/learning-centers/gene-function/viral-transduction/lentivirus/lenti-x-faqs">source</a>)</p>


      
    </div><!-- .post-content -->
  </article><!-- .post -->
  

  
  <nav class="pagination">
    <h2 class="screen-reader-text">Posts navigation</h2>
    <div class="nav-links">
      
      <a href="/page5/" class="newer-posts">&larr; Newer Posts</a>
      
      <span class="page-number">Page 6 of 10</span>
      
      <a href="/page7/" class="older-posts">Older Posts &rarr;</a>
      
    </div>
  </nav><!-- .pagination -->
  

</main><!-- .site-main -->
      <footer class="site-footer">
  <div class="offsite-links">
    
  </div><!-- .offsite-links -->
  <div class="footer-bottom">
    <div class="site-info">
      <p>Copyright © 2007-21 To Sing Fung<br />Powered by Jekyll. Theme modified from Scriptor</p>

    </div><!-- .site-info -->
  </div><!-- .footer-bottom -->
</footer><!-- .site-footer -->

    </div><!-- .inner -->
  </div><!-- .site -->
  
  <!-- Scripts -->
  <script src="/assets/js/plugins.js"></script>
  <script src="/assets/js/custom.js"></script>

</body>
</html>