<!DOCTYPE html>
<html lang="en-US">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Martin Fung&#39;s Page</title>
  <meta name="description" content="">
  <link rel="canonical" href="http://localhost:4000/page10/">
  <link rel="alternate" type="application/rss+xml" title="Martin Fung&#39;s Page Feed"
    href="http://localhost:4000/feed.xml">
  
  <link rel="shortcut icon" href="/favicon.png" type="image/png" />
  
  <!-- Styles -->
  <link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700,700i%7CNoto+Serif:400,400i,700,700i&display=swap" rel="stylesheet">
  <link href="/assets/css/style.css" rel="stylesheet">
</head>

<body>

  <div id="page" class="site">
    <div class="inner">
      <h1 style="text-align: center; padding: 1em 0 0.3em">Martin Fung's Page
</h1>

<h2 style="text-align: center; margin: 0">
    <span>| <a href="/">Home</a></span>
    
    
    
    <span>|</span>
    <a class="" href="/tags/">Blog</a>
    
    
    
    <span>|</span>
    <a class="" href="/cv">CV</a>
    
    <span>|</span>
</h2>
<hr>

      <main class="main-content fadeInDown delay_075s">

  <article class="post" style="border: none; margin-bottom: 8em">
  <h2>Hello!</h2>
  <ul>
  <li>I'm Martin Fung, a biologist from Hong Kong.</li>
  <li><a href="https://pubmed.ncbi.nlm.nih.gov/?term=To+Sing+Fung%5Bau%5D+or+24987391%5Bpmid%5D">Click here for my PubMed search page</a></li>
  <li><a href="https://scholar.google.com/citations?user=igL5v0QAAAAJ">Click here for my Google Scholar page</a></li>
  </article>

  <div style="
  background:#f0f0f0;
  text-align:center;
  padding: 1em 0 1em;">
  <h2 style="margin: 0">
  - Recent posts -</h2>
  </div>
  
  
  <article class="post">
    <header class="post-header">
      <time class="post-date" datetime="2008-08-20">August 20, 2008</time>
      <h2 class="post-title"><a href="/2008-SURF-report" rel="bookmark">2008 SURF report</a></h2>
      <div class="post-meta">
        <span
          class="post-tags"> Tags: 
          <a href="/tags/#undergraduate" rel="tag">undergraduate</a></span>
      </div><!-- .post-meta -->
      
    </header><!-- .post-header -->
    <div class="post-content">
      
      <h2 id="the-role-of-ubiquitin-proteasome-system-in-the-differentiation-of-mouse-embryonic-stem-cell-into-cardiomyocyte">The Role of Ubiquitin Proteasome System in the Differentiation of Mouse Embryonic Stem Cell into Cardiomyocyte</h2>

<p>Student: Martin To Sing Fung<br />Supervisor: Professor Raymond J. Deshaies, Ph.D. <br />Co-mentor: Narimon Honarpour, M.D., Ph.D.</p>

<h2 id="abstract">Abstract</h2>

<p>Cardiovascular disease is the leading cause of death in the world. Present treatments are palliative and patients have poor prognosis. Embryonic stem cell (ES cells) therapy is a promising curative therapy. We study how the differentiation of mouse ES cells into cardiomyocytes is regulated by the ubiquitin proteasome system (UPS), the major mechanism of extra-lysosomal protein degradation. A reporter ES cell line expressing a green fluorescent protein (GFP) gene regulated by the promoter for α-myosin heavy chain (a cardiomyocyte specific gene) is used. Cells are pre-cultured in Col-IV flasks for four days prior to plating, which has been shown to enhance differentiation and assay homogeneity in the pilot experiment. Plated cells are transfected with a siRNA library targeting genes specific to the UPS. GFP signal is screened every other day starting from Day7. We have identified seven potential UPS genes that down regulates cardiomyocyte specific differentiation of ES cells.</p>

<hr />

<h2 id="background">Background</h2>

<p>Cardiovascular disease is the leading cause of death in the world. Congestive heart failure (CHF) accounts for a significant portion of heart disease and its prevalence increases steeply with age. Current therapies for advanced CHF are palliative. In most cases, patients have poor prognosis and the median survival time is 6 months. Heart transplantation is the only cure, but it’s not available for all patients.</p>

<p>One alternative treatment is the transplantation of isolated cardiomyocytes, which has been shown to restore cardiac function in animal models ( <sup id="fnref:1" role="doc-noteref"><a href="#fn:1" class="footnote" rel="footnote">1</a></sup> and <sup id="fnref:2" role="doc-noteref"><a href="#fn:2" class="footnote" rel="footnote">2</a></sup> ). However, the use of neonatal cardiomyocytes is not ethical in human therapies. In order to generate enough cell number, an alternative cell source is ES cells, which can differentiate into cells of all three germ layers. Transplantation of cardiomyocytes derived from human ES cells is shown to restore cardiac function of rats <sup id="fnref:3" role="doc-noteref"><a href="#fn:3" class="footnote" rel="footnote">3</a></sup>. But in clinical application, a pure cardiomyocyte lineage must be selected before transplanted into human body.</p>

<p>The in vitro differentiation of ES cells into cardiomyocytes is regulated by several signaling pathways <sup id="fnref:4" role="doc-noteref"><a href="#fn:4" class="footnote" rel="footnote">4</a></sup>. Inhibitory Wnt signals from the ectoderm and stimulating BMP-2 signals from the endoderm induce mesodermal cells to differentiate into cardiac progenitor cells ( <sup id="fnref:5" role="doc-noteref"><a href="#fn:5" class="footnote" rel="footnote">5</a></sup> and <sup id="fnref:6" role="doc-noteref"><a href="#fn:6" class="footnote" rel="footnote">6</a></sup> ). Cardiac progenitor cells express cardiac specific transcription factors (such as GATA-4, Nkx-2.5), which induce expression of cardiac specific genes. Differentiated cardiomyocytes are structural and functional specialized cells.</p>

<p>The intracellular concentration and activity of these signaling molecules and/or transcriptional factors are regulated by various mechanisms. Ubiquitin-proteasome system (UPS) has been identified as the cell’s major mechanism of extra-lysosomal cytosolic and nuclear protein degradation. It is involved in the regulation of many cellular processes, including cell cycle progression, signal transduction, endocytosis and so on <sup id="fnref:7" role="doc-noteref"><a href="#fn:7" class="footnote" rel="footnote">7</a></sup>. Target proteins are ubiquitinated by three enzymes: ubiquitin-activating enzyme (E1), ubiquitin conjugating enzyme (E2), and ubiquitin protein ligase (E3). E1 activates ubiquitin by ATP hydrolysis, E2 transfers activated ubiquitin to E3, and E3 catalyzes covalent linkage of ubiquitin to the protein substrate. Substrate specificity is mainly provided by hundreds of E3 ligase and other associated adaptor proteins. Poly-ubiquitinated proteins are recognized by the 26S proteasome. The 19S regulatory complex deubiquitinates, unfolds and internalizes the substrate, and the 20S catalytic complex exerts proteolytic degradation.</p>

<p>Recent studies show that the UPS is involved in the regulation of pluripotency maintenance and lineage commitment of ES cells ( <sup id="fnref:8" role="doc-noteref"><a href="#fn:8" class="footnote" rel="footnote">8</a></sup> and <sup id="fnref:9" role="doc-noteref"><a href="#fn:9" class="footnote" rel="footnote">9</a></sup> ). We are interested in how the UPS regulate differentiation of ES cells into cardiomyocytes and design a reporter assay to screen for potential UPS genes.</p>

<h2 id="es-cell-reporter-cell-line-and-sirna-screen-assay-design">ES cell reporter cell line and siRNA screen assay design</h2>

<p>A mouse ES cell line expressing a green fluorescent protein (GFP) gene regulated by the promoter for α-myosin heavy chain (α-MHC) is used. α-MHC is a cardiomyocyte specific gene, thus the GFP expression level indicates the degree of cardiomyocyte specific differentiation of ES cells. This ES cell line can be grown on gelatin-coated substrate, and requires no feeder layer.</p>

<p>ES cells are first cultured in leukemia inhibitory factor (LIF) containing medium, to maintain pluripotency and generate enough cell number. The cells are then plated on gelatin coated 384-well plates, with initial density of about 5500 cells/well. On day2 post-plating, cells in the control set is mock transfected, and cells in the treatment set is transfected with siRNA. The siRNA library targets about 1000 genes specific to the UPS, including E1, E2, E3 and proteasome genes. The majorities are E3 ligase and associated adaptor proteins. The siRNA library is a pooled library: in each well there are 4 siRNA with different sequence target the same UPS gene. GFP expression level in each well is screened at various time points to quantify the degree of carduomyocyte specific differentiation.</p>

<h2 id="optimization-of-assay-condition">Optimization of assay condition</h2>

<p>Prior to the siRNA screen, culturing conditions for spontaneous differentiation of ES cells need to be established, and a pilot experiment was performed in which cells are plated but not treated with siRNA (control set).  Previous tudies show that preculturing ES cells in Collagen-IV (Col-IV) can enhance differentiation <sup id="fnref:10" role="doc-noteref"><a href="#fn:10" class="footnote" rel="footnote">10</a></sup>. Thus the pilot experiment also incorporated two other sets: the Col-IV pre-culture set and the gelatin pre-culture set, in which ES cells were pre-cultured in Col-IV and gelatin respectively for 4 days before plated.</p>

<p>In all the 3 sets of the pilot experiment, cells were plated on 384-well plates as well as slide flasks. The slide flasks were fixed on Day1, Day8 and Day12 post-plating and GFP expression was qualitatively assayed using a fluorescent microscope. The 384-well plates were screened for GFP expression on Day7, Day10 and Day13 post-plating. The images of slide flask pilot experiment is shown as in Table-1. In the control set, no GFP was expressed on Day1 and Day 8. Very low level of GFP was expressed by scattered singlular cells on Day12. 2. In the Col-IV pre-culture set, no GFP was expressed on Day1. Intermediate GFP expression was observed on Day8. On Day12, the GFP intensity was much higher and GFP positive cell areas were extensively distributed on the slide. 3. In the gelatin pre-preculture set, no GFP was expressed on Day1 and Day 8. Intermediate GFP expression was observed on Day12.</p>

<p><img src="https://raw.githubusercontent.com/tosingfung/images/master/surf-1.png" alt="" /></p>

<h4 id="table-1-gfp-expression-of-αmhc-gfp-cells-in-slide-flask-experiment"><strong>Table 1.</strong> GFP expression of αMHC-GFP cells in slide flask experiment</h4>

<p>As shown in Table-2, the results of 384-well plate experiment were in consistence with the slide flasks experiment. GFP expression was quantified as total GFP positive areas (square μm) in each well.</p>

<p><img src="https://raw.githubusercontent.com/tosingfung/images/master/surf-2.png" alt="" /></p>

<h4 id="table-2-gfp-expression-of-αmhc-gfp-cells-in-384-well-plate-experiment"><strong>Table 2.</strong> GFP expression of αMHC-GFP cells in 384-well plate experiment</h4>

<p>To further characterize GFP distribution on a plate, a histogram showing the well number count of different GFP expression level was plotted for Day13 data of the two pre-culture sets, as shown in Figure-1. In the gelatin pre-culture set, most wells had GFP expression in the low to intermediate range. In the Col-IV preculture set, GFP expression was naturally distributed in the intermediate to high range.</p>

<p><img src="https://raw.githubusercontent.com/tosingfung/images/master/surf-3.png" alt="" /></p>

<h4 id="figure-1-well-number-count-of-total-gfp-positive-areas-in-col-iv-pre-culture-day-13-plates-and-gelatin-pre-culture-day-13-plates"><strong>Figure 1.</strong> Well number count of total GFP positive areas in Col-IV pre-culture Day 13 plates and gelatin pre-culture Day 13 plates</h4>

<p>To conclude the pilot experiments, ES cells plated on gelatin coated substrate without pre-culture have very limited cardiomyocyte specific differentiation. Pre-culturing ES cells in gelatin can enhance cardiomyocyte differentiation to a certain degree. Pre-culturing ES cells in Col-IV enhances differentiation into cardiomyocytes significantly, resulting in a high level and homogenous expression of GFP. Thus, Col-IV pre-culture was incorporated into the siRNA screen protocol.</p>

<h2 id="sirna-screening-assay">siRNA screening assay</h2>

<p>Based on the pilot experiment, the established culture condition was applicable and thus utilized in the siRNA screen. After 4 day pre-culture in Col-IV, ES cells were plated on twelve 384-well plates. On day2 post-plating, ten plates with most similar cell density were chosen and cells were transfected.</p>

<p>The assay was done duplicated; all wells in Plate 1, 2, 3, 4, 1R, 2R, 3R and 4R were treated with siRNA from the library. Four rows in Plate 5 and 5R were transfected with siRNA, the remaining wells were mock transfected and served as the control. GFP expression was screened on day4, 6, 8, 10, 13 and 15 post-plating. Before day10, GFP expression was insignificant. After day10, cell detachment was observed in some over-confluent wells. Thus, Day10 data was analyzed. Table-3 shows the data analysis of all ten plates and the control set.</p>

<p><img src="https://raw.githubusercontent.com/tosingfung/images/master/surf-4.png" alt="" /></p>

<h4 id="table-3-gfp-expression-on-ten-plates-and-the-control-set-square-micron"><strong>Table 3.</strong> GFP expression on ten plates and the control set (square micron)</h4>

<p>There was significant variation within a plate, shown by the large standard deviation in all plates and in the non-treated control sets. Two plates, plate 2R and plate 4, had significant larger mean compare to the control sets and other plates. However, this was unlikely caused by siRNA treatment, because it was not observed in the replica: plate 2 and 4R. Thus the high GFP expression was most likely due to high initial cell density in these two plates. All other plates behaved similarly to the control. Thus we can assume that, on any transfected plate, the net effect of siRNA treatment in all wells was close to zero. Thus, the mean value of each plate can be used as an internal standard for its own wells.  Based on this, we established a scheme to score GFP expression in the wells:</p>

<table>
  <thead>
    <tr>
      <th>GFP expression of a well</th>
      <th>Score</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Between “mean + SD” and “mean + 2SD” of its own plate</td>
      <td>1</td>
    </tr>
    <tr>
      <td>Between “mean + 2SD” and “mean + 3SD” of its own plate</td>
      <td>2</td>
    </tr>
    <tr>
      <td>Above “mean + 3SD” of its own plate</td>
      <td>3</td>
    </tr>
  </tbody>
</table>

<p>The possibility of a well being a potential positive hit is given by the sum score in the same well on both a plate (plate#) and its replicate (plate#R):</p>

<table>
  <thead>
    <tr>
      <th>Possibility of a well being a positive hit</th>
      <th>Sum score (Plate # + Plate # R)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>High</td>
      <td>6</td>
    </tr>
    <tr>
      <td>Intermediate to high</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Intermediate</td>
      <td>4</td>
    </tr>
    <tr>
      <td>Low to intermediate</td>
      <td>3</td>
    </tr>
    <tr>
      <td>Low</td>
      <td>2</td>
    </tr>
  </tbody>
</table>

<p>Imposing this scoring scheme on the day10 data, one “intermediate to high” and six “intermediate” hits were identified. Based on preliminary review, these hits corresponded to either E3 ligase or associated adaptor proteins. We still need to analyze data of other time points to confirm the result.</p>

<h2 id="validating-experiments">Validating experiments</h2>

<p>To validate the potential positive hits identified in the screen, the corresponding pooled siRNA will be separated into four individual siRNA and re-screen with the cells. Within four wells targeting the same UPS gene, if 3-4 wells are still positive, then the potential positive hit will be confirmed as true hits. However, if 0-2 wells are positive, the potential positive hit will be rejected. The true hits will be further validated with another ES cell reporter cell line, which expresses a luciferase gene regulated by the α-MHC promoter.</p>

<h2 id="sirna-screening-assay-limitation-and-improvement">siRNA screening assay limitation and improvement</h2>

<p>We discovered some limitations in the siRNA screen assay. First, siRNA transfection is a stress for ES cells. It inhibits ES cells’ proliferation and differentiation. There is no alternative method to deliver the siRNA. To compensate for the stress, GFP screens at early time points (Day4, 6, 8) can be omitted. Thus, the period that cells stay outside the incubator is minimized. Secondly, there is a large inter-plate deviation. Thus, control wells should be introduced into each plate as internal standard. Finally, the effect of siRNA only last for 1-2 days. Thus the assay only screen for genes that act within the 2 days post-transfection. Alternatively, other silencing agent such as shRNA can be used to have a more sustained silencing effect.</p>

<h2 id="conclusion">Conclusion</h2>

<p>We have established a cell-based siRNA screen assay to identify potential UPS genes that regulate differentiation of ES cells into cardiomyocytes. Pre-culturing ES cells in Col-IV can enhance cardiomyocyte differentiation and assay homogeneity. Seven potential UPS genes (E3 ligases and associated adaptor proteins) are identified. To confirm the result, further data analysis and re-screen are required.</p>

<h2 id="acknowledgement">Acknowledgement</h2>

<p>I would like to thank Professor Raymond J. Deshaies for accepting me as SURF student and all his help through the project. I would also like to thank Narimon Honarpour for teaching me everything about the project and all the techniques involved. My summer research was sponsored by Caltech Alumni Association Hongkong Chapter.</p>

<h2 id="reference">Reference</h2>

<div class="footnotes" role="doc-endnotes">
  <ol>
    <li id="fn:1" role="doc-endnote">
      <p>Müller-Ehmsen, J. et al. Survival and Development of Neonatal Rat Cardiomyocytes Transplanted into Adult Myocardium. Journal of Molecular and Cellular Cardiology. Vol 34, Issue 2, 107-116 (2002). <a href="#fnref:1" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:2" role="doc-endnote">
      <p>Reffelmann, T. et al. Transplantation of neonatal cardiomyocytes after permanent coronary artery occlusion increases regional blood flow of infarcted myocardium. Journal of Molecular and Cellular Cardiology. Vol 35, Issue 6, 607-613 (2003). <a href="#fnref:2" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:3" role="doc-endnote">
      <p>Laflamme1, M. et al. Cardiomyocytes derived from human embryonic stem cells in pro-survival factors enhance function of infarcted rat hearts. Nature Biotechnology 25, 1015 - 1024 (2007). <a href="#fnref:3" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:4" role="doc-endnote">
      <p>Solloway, M., Harvey, R. Molecular pathways in myocardial development: a stem cell perspective. Cardiovascular Research. Vol 58, Issue 2, 264-277 (2003). <a href="#fnref:4" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:5" role="doc-endnote">
      <p>Ueno, S. et al. Biphasic role for Wnt/beta-catenin signaling in cardiac specification in zebrafish and embryonic stem cells. PNAS. Vol 104, Issue 23, 9685-9690 (2007). <a href="#fnref:5" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:6" role="doc-endnote">
      <p>Andree, B. et al. BMP-2 induces ectopic expression of cardiac lineage markers and interferes with somite formation in chicken embryos. Mechanisms of Development. Vol 70, Issue 1-2, 119-131 (1998). <a href="#fnref:6" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:7" role="doc-endnote">
      <p>Hershko, A. et al. The Ubiquitin System. Annual Review of Biochemistry. Vol 67, 425-479 (1998). <a href="#fnref:7" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:8" role="doc-endnote">
      <p>Naujokat, C., Saric T. Concise review: role and function of the ubiquitin-proteasome system in mammalian stem and progenitor cells. Stem Cells. Vol 25, 2408-2418 (2007). <a href="#fnref:8" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:9" role="doc-endnote">
      <p>Westbrook, T. et al. SCF β-TRCP controls oncogenic transformation and neural differentiation through REST degradation. Nature. Vol 452, 370-375 (2008). <a href="#fnref:9" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:10" role="doc-endnote">
      <p>Schenke-Layland, K. et al. Collagen IV induces trophoectoderm differentiation of mouse embryonic stem cells. Stem Cells. Vol 25, 1529-1538 (2007). <a href="#fnref:10" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
  </ol>
</div>

      
    </div><!-- .post-content -->
  </article><!-- .post -->
  
  <article class="post">
    <header class="post-header">
      <time class="post-date" datetime="2007-08-20">August 20, 2007</time>
      <h2 class="post-title"><a href="/2007-DREAM-report" rel="bookmark">2007 DREAM report</a></h2>
      <div class="post-meta">
        <span
          class="post-tags"> Tags: 
          <a href="/tags/#undergraduate" rel="tag">undergraduate</a></span>
      </div><!-- .post-meta -->
      
    </header><!-- .post-header -->
    <div class="post-content">
      
      <h2 id="microfluidic-and-microbiology">Microfluidic and Microbiology</h2>

<p>Student: Martin To Sing Fung<br />Supervisor: Professor Leo Lau</p>

<h2 id="background">Background</h2>

<p>In 2005, it was reported that pilus-like appendages are produced by Shewanella oneidensis in direct response to electron acceptor limitation. The apparent height of these appendages in scanning tunneling microscope (STM) is about 10nm, which is in consistent with scanning electron microscope (SEM) measurement, indicating that they are electrical conductive. (In STM, conducting samples should have apparent height close to actual height. If a sample feature is not conductive, the feature is invisible to STM and the apparent height should be zero. ) These appendages are thus named bacterial nanowires. When S.oneidensis is grown with silica ferrihydrite as the only electron acceptor, nanocrystalline magnetite is found arranged in linear arrays, whose dimensions consistent with nanowires.(Figure.1) It is proposed that nanowires are used to transport electrons to extracellular solid iron oxide.</p>

<p><img src="https://raw.githubusercontent.com/tosingfung/images/master/dream1.png" alt="dream1" /></p>

<h4 id="figure-1">Figure 1</h4>

<p>Another group studied on anaerobic bacteria Geobacter sulfurreducens, which produce nanowires to attach to and reduce Fe (III). Using conducting probe atomic force microscopy (CP-AFM), the nanowires were also found highly conductive. Certain pilus-deficient mutants could not reduce Fe (III) but could attach to them with nanowires. (Lovley, 2005) However, some skeptics claimed that the TEM image (Figure.1) is a thin section of sample, and for three dimensionally protruding nanowires, it’s fairly unlikely that the section will cut through three nanowires at the same time. So, these linear structures are more likely to be extracellular polymeric substances secreted by the bacteria. Also, in Lovley’s CP-AFM study, the measured part between the conductive tip and graphite support is the equatorial section of the pilus filament, so whether it is conductive along the nanowire is not tested.</p>

<h3 id="experimental-design-and-theory">Experimental design and theory</h3>

<p>Using an alternative approach, we have the idea of inducing the nanowire to grow in a confined nanochannel in a microfluidic device. The design is shown in Figure.2, which is embedded between two glass slides. To save cost, two separate systems are put on one device. In each system, the large circles at top and bottom are the inlets and outlets. Thick lines are microchannels where bacteria culture or mineral solution flows through, which are about 25μm wide. The two smaller circles in the middle indicate chambers, which are not to scale. The diameter should be around 60μm. The thin lines between two chambers refer to the nanochannels, each of which is 400nm wide and deep, about 25μm long. It is slightly narrower than the diameter of S.oneidensis cells; also given the chemotaxis motion (thrust and tumble) of bacteria, it is fairly unlikely that a bacterium will flow through the nanochannels to the mineral chamber. Except for the nanochannels, the depths of all device features are 8.5μm.</p>

<p><img src="https://raw.githubusercontent.com/tosingfung/images/master/dream2.png" alt="dream2" /></p>

<h4 id="figure-2">Figure 2</h4>

<p>Then S.oneidensis is injected in the bacteria chamber and uranyl acetate in the mineral chamber. When oxygen is depleted, S.oneidensis will shift to second best available electron acceptor. Sensing the trace amount of uranyl acetate yet unable to move to the opposite chamber, S.oneidensis may then produce nanowires through interlinking nanochannels. If black precipitation of reduced uranium is observed in mineral chamber, we can crack the device to image and analyze it. Microfluidics provides many advantages in this study. Fluid has unique properties in micro-scale, such as laminar and electro-osmotic flow, which can be utilized to manipulate micro environment of the bacteria chamber. What is more, a microfluidic device can easily combine with micro electro mechanical systems (MEMs). For example, a refined device with probes at the two ends of nanochannel can characterize the electrical properties along the nanowires. A microfluidic device also has the advantage of saving materials. Not only fewer bacteria are needed, but the use of toxic mineral (uranyl acetate) is reduced.</p>

<h3 id="objective">Objective</h3>

<p>If the hypothesis stands, we expect to observe bacterial nanowires in the nanochannels and possibly in the mineral chamber. Coexistence of nanowires and reduced uranium precipitations will be good evidence that nanowires are used to transport electrons and reduce uranyl acetate. If the two chambers are linked with nanowires, there will also be an increase of conductivity along the nanochannels. In confirming that nanowires are produced under electron acceptor limitation and are used by S.oneidensis to transport electrons to extracellular minerals, we can further study the electrical and chemical properties of the nanowires by further modifying the device and combining it with MEMs. We can also study the molecular basic of nanowires’ conductivity and its functioning mechanisms. Moreover, mutant studies will allow us to unravel the genetic and molecular basis of nanowire function.</p>

<h2 id="experimental-design-and-procesures">Experimental design and procesures</h2>

<h3 id="1-fabrication-of-microfluidic-device">1. Fabrication of microfluidic device</h3>

<ol>
  <li>
    <p>A glass slide is cleaned. Using E-beam evaporation, chromium (10nm) is coated as the adhesion layer. Then, another gold layer (150nm) is coated on top. These metal layers serve as media to transfer the mask pattern from the photoresist layer to the glass slide (because glass etchant hydrogen fluoride is not selective and reacts with exposed photoresist, a media layer is required). Also, the chromium layer is essential in the carving of nanochannel with focused ion beam (FIB).</p>
  </li>
  <li>
    <p>Positive photoresist (Shipley® SC1827) is spread (500 rpm, 10 s) and spin-coated (2500 rpm, 45 s) on top of the gold layer. The thickness of photoresist layer is about 3.5 micron.</p>
  </li>
  <li>
    <p>Photolithography. A 5’ by 5’ mask is prepared, with channels and chambers’ pattern transparent and other area opaque. UV light is then shined through the mask onto the glass slide prepared in step2. Chemical reactions will occur in and only in the exposed region of photoresist layer.</p>
  </li>
  <li>
    <p>The exposed region is washed away with photoresist developer (MF-319).</p>
  </li>
  <li>
    <p>Gold etchant is added to transfer pattern to gold layer. After that chromium etchant is added to further transfer pattern to the chromium layer. Both etching last for 30 seconds and are isotropic.</p>
  </li>
  <li>
    <p>Hydrogen fluoride is added to etch microchannels and chambers into the glass slide. This is also isotropic, so the edges are round and actual diameter will be little bigger than that on the mask. After 45 minutes, the pattern is about 8.5 microns deep into glass.</p>
  </li>
  <li>
    <p>Photoresist and gold coating are removed.</p>
  </li>
  <li>
    <p>Using Focused Ion Beam (FIB-SEM), five nanochannels (400nm deep and wide, about 25 micron long) is fabricated between two chambers. Gallium ion is used to sputter the surface and dry-etch into glass. Because focused ion beam requires a conductive layer (so that positive charge will not accumulate and eventually repel positive gallium ion), the chromium layer is not washed in step 7. Also, SEM is used for accurate positioning of nanochannels, which also needs a conductive surface.</p>
  </li>
  <li>
    <p>The chromium layer is removed.</p>
  </li>
  <li>
    <p>Cover glass bonding. Because the device has to be cracked open after the experiment, permanent bonding can’t be used. Also, ordinary microscopy cover glass is too flexible to withstand the pressure of injection. Therefore, we use anther glass slide as cover glass. Eight holes are drilled beforehand at the location of the inlets and outlets. The bonding surfaces are cleaned, moisturized and directly bonded. Cover glass is essential in making the device completely isolated from atmosphere and creates the anaerobic environment that is critical for nanowire production.</p>
  </li>
  <li>
    <p>Flat bottom ports are added on top of all inlets and outlets. These ports are convenient adaptors which keeps device anaerobic yet provide easy access whenever injection of fresh medium/bacteria/minerals is necessary.</p>
  </li>
</ol>

<p>Figure.3 shows what a complete device looks like (A, B). 3-C illustrate the relative dimension of chambers, nanochannels and S.oneidensis cells under SEM. 3-D is the unfilled device under an optical microscope. When the microchannels and chambers are filled up, it becomes transparent, so do the nanochannels shown in the middle.</p>

<p><img src="https://raw.githubusercontent.com/tosingfung/images/master/dream3.png" alt="dream3" /></p>

<h4 id="figure-3">Figure 3</h4>

<h3 id="2-growing-of-s-oneidensis-culture">2. Growing of S. oneidensis culture</h3>

<p>Shewanella oneidensis is a gram negative bacterium. It is widely used as the model organism of metal reducing bacteria due to the following reasons:</p>

<ol>
  <li>
    <p>Metabolic robust. S.oneidensis can utilize various kinds of electron donor (lactate, acetate, pyruvate, serine, and other amino acids) and electron acceptors (S, S2O32-, NO3-, NO2-, trimethylamine oxide, dimethyl sulfoxide, fumarate, soluble and insoluble forms of iron (III), manganese (III/IV) etc.).</p>
  </li>
  <li>
    <p>Facultative anaerobic. S.oneidensis can live with or without oxygen. This makes culturing easy and incurs research interest in the genetic switching basis behind this environmental responds. This property is beneficial for our experiment because trace amount of oxygen within the device can be used for bacterial growth and replication; whereas the anaerobic condition afterwards will induce electron acceptor to shift and production of nanowires.</p>
  </li>
  <li>
    <p>The genome of S.oneidensis has been mapped, which makes mutant study possible. By knocking out or overexpressing certain genes, we can unravel the molecular basis of nanowire function. S.oneidensis from rich medium is inoculated into a minimal defined medium of the following chemical compositions:</p>
  </li>
</ol>

<table>
  <thead>
    <tr>
      <th>Constituents</th>
      <th>Concentration</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>sodium lactate</td>
      <td>18 mM/L</td>
    </tr>
    <tr>
      <td>PIPES buffer</td>
      <td>3 mM/L</td>
    </tr>
    <tr>
      <td>sodium bicarbonate</td>
      <td>20 mM/L</td>
    </tr>
    <tr>
      <td>ammonium chloride</td>
      <td>28 mM/L</td>
    </tr>
    <tr>
      <td>sodium biphosphate</td>
      <td>4.35 mM/L</td>
    </tr>
    <tr>
      <td>nitrilotriacetic acid</td>
      <td>0.1 mM/L</td>
    </tr>
  </tbody>
</table>

<p>Sodium lactate is the electron donor for S.oneidensis; it also serves as the carbon source in assimilation. PIPEs buffer maintains the medium at around pH 6.8, and it makes medium isotonic to bacterial cytoplasm, so that cells will not burst due to water inflow under osmotic pressure. Sodium bicarbonate also works as buffer. Ammonium chloride and NaH2PO4 are nitrogen and phosphate source respectively. Nitrolotriacetic acid is a chelator of metal ions. It can shuttle metal ions to and fro the two chambers in its alternative redox forms. Vitamins and minerals are essential for bacterial survival because they are necessary components of enzymes or coenzymes. Final pH was adjusted to 7.0 with 2M HCl. A defined minimal medium is used because it contains and only contains all the chemicals necessary for S.oneidensis survival. Also, under anaerobic condition, no chemical in the mineral medium can serve as an alternative electron acceptor, so S.oneidensis is obliged to produce nanowires in search for uranyl acetate in the mineral chamber.</p>

<h3 id="3-induction-nanowire-growth-in-nanochannel">3. Induction nanowire growth in nanochannel</h3>

<p>The uranyl acetate is kept in anaerobic for two days to get rid of any dissolved oxygen. Then it is injected via a syringe filter into the mineral chamber through the mineral inlet under an optical microscope. When the microchannels and chamber become transparent, plugs are added on the flat bottom ports of mineral inlet and outlet to ensure anaerobic environment. At the time of experiment, the population density of S.oneidensis minimal culture is about 108-109 cells/mL. After injection, around 10 - 50 cells will remain in the bacterial chamber. The bacterial inlet and outlet are plugged as well to ensure anaerobic. The device is then left in a dark and cool bench for a week, during which the bacteria grow and replicate. Fresh medium is regularly supplied. Under optical microscope, the cell number in bacterial chamber increases steadily. After 9 days, black precipitation is observed in the mineral chamber and the device is ready for analysis.</p>

<h3 id="4-sem-imaging-and-further-characterization">4. SEM imaging and further characterization</h3>

<p>Glucoaldehyde is injected into the bacteria chamber to fix all the cells and cellular components. Then the device is cracked open and put in 25% ethanol. Increasing concentration of ethanol (50%, 75% and 100%) is used to gradually dehydrate the sample. The device is then transferred from 100% ethanol to critical point dryer and dried with liquid carbon dioxide. When liquid carbon dioxide completely substitutes ethanol, temperature and pressure are adjusted so that carbon dioxide reaches critical point, where liquid phase and gaseous phase are identical. Since there is no surface tension, the delicate structures are well preserved. The sample is then coated with platinum for SEM imaging. As mentioned above, coexist of bacterial nanowire and reduce uranium is a strong evidence to support the hypothesis. But the dimension of nanowires are too small to identify using SEM. Alternatively, we can use TOF-SIMs to analyze elemental composition of the nanochannel. Since cytochromes play an important role in electron transport, they are believed to be major conductive component of bacterial nanowires. Cytochromes are heme-bound proteins, thus coexist of iron and uranium in the nanochannel will be a good indication. Unfortunately, we don’t have enough time to do this on a cracked device.</p>

<p><img src="https://raw.githubusercontent.com/tosingfung/images/master/dream4.png" alt="dream4" /></p>

<h4 id="figure-4">Figure 4</h4>

<p>Figure.4 shows the SEM images token of a cracked device. Sphere-like structures are observed all around the device. The best guess is they are uranyl phosphate precipitates. When device is cracked open, uranyl acetate in the mineral chamber and phosphate in the bacterial chamber mix and form these insoluble spheres. Regardless of this, the SEM results contain features we expected to see and strongly support an electron conducting role of nanowires. In A and B, we can see all five nanochannels are partially or entirely filled up with filaments. The first one from the right are entirely filled and at the side of bacterial chamber, a micro colony of S.oneidensis is found. C is a closer look at the nanochannels. The egg-like structure in the middle is predicted to be reduced uranium (IV) precipitation. Figure.4D is a closer look at the mineral chamber. Reduced uranium (IV) precipitations spread all around. Numerous filaments attach to precipitated particles in a web-like manner, which is similar to the TEM image.</p>

<h2 id="discussion">Discussion</h2>

<h3 id="drawback-of-the-current-design">Drawback of the current design</h3>

<ol>
  <li>
    <p>Because the device has to be cracked open for analysis, permanent bonding can not be used. Direct bonding of two glass slide requires a very clean surface and takes a long time of trial and error. Also, sometimes leakage will occur and the device has to be cracked and bond again.</p>
  </li>
  <li>
    <p>Ordinary microscopic cover glass is much thinner than the device’s cover glass. They are not used because they are too flexible to withstand direct bonding and pressure of injection. Since thick cover glass is used, microscope with short working distance can not be used.</p>
  </li>
  <li>
    <p>Direct bonding creates a vacuum between two glass slides. When the device is cracked, contents in the chamber will spread everywhere on the glass slide. Also, delicate structure such as nanowires may be damaged.</p>
  </li>
  <li>
    <p>Under aerobic condition, S.oneidensis is highly adhesive to the device surface because biofilm is formed. However, when oxygen is depleted, the bacteria tend to detach and flow away.</p>
  </li>
  <li>
    <p>After each experiment, some organic remains are secreted by the bacteria, which stick to the device and can’t be removed even by strong acid like aqua regia and nitrous acid, neither by strong detergent like SDS.</p>
  </li>
</ol>

<h3 id="improvements">Improvements</h3>

<p>We make some improvements regarding the above mentioned problems:<br /></p>
<ol>
  <li>
    <p>Pyrex borosilicate glass is used as cover glass instead. This glass has the property of being strong and thin at the same time. It can substitute ordinary glass slide as cover glass while at the same time only half as thick (5mm).</p>
  </li>
  <li>
    <p>Using PDMS as cover layer. Pyrex can not solve the problem of drastic cracking process. PDMS, which is a biocompatible, near transparent polymer, is flexible but bond to glass tightly. Because it is elastic, the cracking simply means peeling away PDMS layer. This gentle process will not spread chamber contents or damage microstructures.</p>
  </li>
  <li>
    <p>Concerning the SEM artifacts, next time before the device is cracked; fresh water should be used to flush both chambers. Uranium precipitate and bacterial biofilm won’t be easily flushed away, whereas the soluble uranium acetate and sodium phosphate concentration will greatly decrease. Thus even they mix afterwards; large uranyl phosphate precipitate will not form.</p>
  </li>
</ol>

<h2 id="conclusion">Conclusion</h2>

<p>Though this experiment doesn’t provide direct evidence that S.oneidensis produce conductive nanowires to reduce uranyl acetate, the preliminary SEM results show that nanowires are produced under oxygen limited condition, and they are highly associated with uranyl precipitation. This indicates bacterial nanowires’ critical function in mineralization. And it proves that microfluidic device can be used microbiology research; and should be used due to its numerous advantages. Although problems reside in the practical manipulation, this experiment is a pioneer in trying to combine microfluidic techniques and microbiology study, and it provides insights and new prospective in biomineralization research.</p>

<h2 id="reference">Reference</h2>

<ol>
  <li>
    <p>Yuri A. Gorby et al. Electrically conductive bacterial nanowires produced by Shewanella oneidensis strain MR-1 and other microorganisms. PNAS. July 25, 2006. Vol. 103. No. 30.</p>
  </li>
  <li>
    <p>Derek R. Lovley et al. Extracellular electron transfer via microbial nanowires. Nature. June 23, 2005. Vol 435.</p>
  </li>
</ol>

      
    </div><!-- .post-content -->
  </article><!-- .post -->
  

  
  <nav class="pagination">
    <h2 class="screen-reader-text">Posts navigation</h2>
    <div class="nav-links">
      
      <a href="/page9/" class="newer-posts">&larr; Newer Posts</a>
      
      <span class="page-number">Page 10 of 10</span>
      
    </div>
  </nav><!-- .pagination -->
  

</main><!-- .site-main -->
      <footer class="site-footer">
  <div class="offsite-links">
    
  </div><!-- .offsite-links -->
  <div class="footer-bottom">
    <div class="site-info">
      <p>Copyright © 2007-21 To Sing Fung<br />Powered by Jekyll. Theme modified from Scriptor</p>

    </div><!-- .site-info -->
  </div><!-- .footer-bottom -->
</footer><!-- .site-footer -->

    </div><!-- .inner -->
  </div><!-- .site -->
  
  <!-- Scripts -->
  <script src="/assets/js/plugins.js"></script>
  <script src="/assets/js/custom.js"></script>

</body>
</html>